### PR TITLE
bohra 1.0.10 (new recipe)

### PR DESCRIPTION
**Pull request by tseemann**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by tseemann** [comment]
WIsh us luck @/kristyhoran  !

File: recipes/bohra/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - python >=3.7
+    - biopython >=1.70
+    - pathlib
+    - numpy
+    - svgwrite
+    - psutil
+    - pytest
+    - sh
+    - packaging
+    - datetime
```
Comment (Slugger70) [inline recipes/bohra/meta.yaml]: Is this in the standard python 3.7 library?

Comment (tseemann) [reply recipes/bohra/meta.yaml]: I can only see `datetime` in the standard library list: https://docs.python.org/3/library/

File: recipes/bohra/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - shovill
+    - roary
+    - prokka
+    - kraken2
+    - abricate
+    - mlst
+    - iqtree
+    - seqtk
+    - snp-dists
+    - zlib
```
Comment (bgruening) [inline recipes/bohra/meta.yaml]: Why does zlib needs to be here, if you do not complile anything?

**Comment by bgruening** [comment]
pytest-runner seems to be a dependency that needs to be added.

**Comment by tseemann** [comment]
I am deleting this PR because the author @kristyhoran is taking over.

**Pull request closed without merging**